components/avatar/avatar.css

```diff
@@ -7,25 +7,28 @@
   block-size: auto;
   aspect-ratio: 1/1;
 }
-.avatar-small img {
+.avatar-with-image.small img {
   border: 2px solid var(--primary-color-dark);
 }
-.avatar-medium img {
+.avatar-with-image.medium img {
   border: 3px solid var(--primary-color-dark);
 }
-.avatar-large img {
+.avatar-with-image.large img {
   border: 4px solid var(--primary-color-dark);
 }
-.avatar-small {
+.avatar-with-image.small,
+.avatar-with-text.small {
   width: 50px;
   height: 50px;
 }
 
-.avatar-medium {
+.avatar-with-image.medium,
+.avatar-with-text.medium {
   width: 100px;
   height: 100px;
 }
-.avatar-large {
+.avatar-with-image.large,
+.avatar-with-text.large {
   width: 150px;
   height: 150px;
```

Comment:
I think rem should be preferable here

Reply:
I forgot to change to rem in the final version. Thank you for pointing that out.